tests/e2e/tidbcluster/across-kubernetes.go

```diff
@@ -401,8 +401,6 @@ var _ = ginkgo.Describe("[Across Kubernetes]", func() {
 			tc1 := GetTCForAcrossKubernetes(ns1, tcName1, version, clusterDomain, nil)
 			tc2 := GetTCForAcrossKubernetes(ns2, tcName2, version, clusterDomain, tc1)
 			tc3 := GetTCForAcrossKubernetes(ns3, tcName3, version, clusterDomain, tc1)
-			// FIXME(jsut1900): remove this after #4361 get fixed.
```

Comment:
Why skip TiKV in L526?

Reply:
We have failed Tikv before failing PD, though it should make no difference to restart a failed tikv pod.

Reply:
It's different, the first part is to check that Pods can restart successfully after all TiKV down, and the second part is to check that Pods can restart successfully after all PD down.

Reply:
addressed in https://github.com/pingcap/tidb-operator/pull/4382